Classes/Service/NodePolicyService.php

```diff
@@ -82,6 +82,7 @@ public function getNodePolicyInformation(NodeInterface $node): array
             'disallowedNodeTypes' => $this->getDisallowedNodeTypes($node),
             'canRemove' => $this->canRemoveNode($node),
             'canEdit' => $this->canEditNode($node),
+            'canView' => $this->canReadNode($node),
```

Comment:
Would prefer `canRead` to implicate the relationship to the privilege and other methods.

Reply:
done